src/vuex/modules/connection.js

```diff
@@ -34,14 +34,14 @@ export default function({ apollo }) {
         ? data.networks.find(network => network.id === persistedNetwork)
         : false
       if (persistedNetwork && storedNetwork) {
-        await commit(`setNetworkId`, storedNetwork.id)
+        await dispatch(`setNetwork`, storedNetwork)
```

Comment:
why was this necessary?

Reply:
Because this is the proper way, otherwise it fails in execution. But I changed it yesterday fast to pass the tests 😅